superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -444,6 +469,11 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
   const [editNewDb, setEditNewDb] = useState<boolean>(false);
   const [isLoading, setLoading] = useState<boolean>(false);
   const [testInProgress, setTestInProgress] = useState<boolean>(false);
+  const [passwords, setPasswords] = useState<Record<string, string>>({});
+  const [confirmedOverwrite, setConfirmedOverwrite] = useState<boolean>(false);
+  const [file, setFile] = useState<UploadFile[]>([]);
```

Comment:
Can we rename this back to `fileList` and `setFileList`, so we know that it should indeed be a list of files (even if it will have only one)? It also matches the name of the prop in `<Upload/>`.

Reply:
+1

Reply:
Whoops, I meant to do this and forgot haha! Good catch, changed in [`this commit`](https://github.com/apache/superset/pull/19314/commits/ba13443eea8fcde7512238f5e0121971dd092e20)